src/Indicator.vala

```diff
@@ -330,6 +335,50 @@ public class Session.Indicator : Wingpanel.Indicator {
         current_dialog.set_transient_for (indicator_icon.get_toplevel () as Gtk.Window);
         current_dialog.show_all ();
     }
+
+    private async void update_tooltip () {
+        string description;
+        string other_users = "";
+
+        if (server_type == Wingpanel.IndicatorManager.ServerType.SESSION) {
+
+            if (active_user == null) {
+                debug ("active_user is null. Getting active user");
+                active_user = yield manager.get_active_user ();
+            }
+
+            string active_real_name;
+            int n_online_users = (yield manager.get_n_active_and_online_users ()) - 1;
+
+            if (active_user == null) {
+                critical ("Active user null. Cannot get real name");
+                active_real_name = _("Unknown");
+            } else {
+                active_real_name = active_user.get_real_name ();
+            }
+
+            description = _("Logged in as %s").printf (active_real_name);
+
+            if (n_online_users > 0) {
+                other_users = _(", %i other %s logged in").printf (
+                    n_online_users,
+                    ngettext (_("user"), _("users"), n_online_users)
+                );
```

Comment:
Maybe need i18n team advice on this but in general it is a bad idea to construct translated strings from parts.  I think this is better:
```
            if (n_online_users > 0) {
                other_users = ngettext (_(", %i other user logged in"),
                                        _(", %i other users logged in"),
                                        n_online_users).printf (n_online_users);
            } else {
                other_users = "";
            }
```